app/src/main/java/com/nononsenseapps/feeder/ui/compose/searchfeed/SearchFeedViewModel.kt

```diff
@@ -27,6 +31,7 @@ import java.time.Instant
 class SearchFeedViewModel(di: DI) : DIAwareViewModel(di) {
     private val feedParser: FeedParser by instance()
     private val repository: Repository by instance()
+    private val application: FeederApplication by instance()
```

Comment:
```suggestion
    private val application: Application by instance()
```

This caused a crash in my tests

Reply:
Fixed.